pom.xml

```diff
@@ -32,11 +32,6 @@
 
     <inceptionYear>2018</inceptionYear>
 
-    <organization>
-        <name>EntIT Software LLC, a Micro Focus company</name>
-        <url>https://www.microfocus.com</url>
-    </organization>
```

Comment:
Why is this being removed instead of updated?

Reply:
I wasn't sure about this, it looks like it hasn't been updated in a long time so must not be in use as EntIT has not been sued for a long time. If its safer I can update this, is this ok?
```
    <organization>
        <name>Open Text</name>
        <url>https://www.opentext.com</url>
    </organization>
```